_includes/known-issues.html

```diff
@@ -1,17 +1,17 @@
-{% assign page_name = page.title | downcase %}
-{% assign page_slug = page_name | slugify %}
-{% assign github_label = page.github_label_url | default: page.title | replace: ' ', '+' %}
-{% assign issues = site.data.issues[page_slug] %}
+{% assign github_label =  page.github_label_url | default: page.title | replace: ' ', '+' %}
+{% assign github_label_alt =  page.known_issues_key | replace: ' ', '+' | replace: '-', '+' %}
 
-{%- if issues.items -%}
-  <h2 id="known-issues" class="site-component-section-title">Known issues</h2>
+<h2 id="known-issues" class="site-component-section-title">Known issues</h2>
 
-  <p>All known issues for {{ page.title | downcase }} are tracked in
-  <a href="https://github.com/uswds/uswds/issues?q=is%3Aissue+is%3Aopen+label%3A%22Package%3A+{{ github_label }}%22">GitHub</a>.
-  Notable usability issues are tracked below: </p>
+<p>All known issues for {{ page.title | downcase }} are tracked in
+  {% if page.known_issues_key %}
+    <a href="https://github.com/uswds/uswds/issues?q=is%3Aissue+is%3Aopen+label%3A%22Package%3A+{{ github_label_alt }}%22%2C%22Package%3A+{{ github_label }}%22">
```

Comment:
>[!note]
> This updates the search query to include labels for both the component itself (date range picker) and the component that has known issues (date picker).

Reply:
Is this comment still relevant? Looks like the code has been updated since

Reply:
It is not! Thanks for catching this. I'll resolve the comment now. Everyone else reading this, kindly disregard my above comment :)